### ISSUE TITLE
Language specific morphemizer

### ISSUE THREAD

**Issue by Vilhelm-Ian** [kaegi/MorphMan#277]
When using morphman for language with spaces K and V are the same because it treats every single word as a different morph. So features like readiabilty analyzer are less useful. And the cards it shows often contains words that are already learned.

**Comment by yaoberh** [comment]
Yes, this is true and it happens with all languages I tried recently.

**Comment by Vilhelm-Ian** [comment]
It's not the case for japanese. It would be nice if someone explained how the dictionary addons that work for japanese work so a similar addon can be made for other languages

**Comment by yaoberh** [comment]
I'm using this addon as analizer for Japanese:
[](https://github.com/CrescentKohana/MeCabUniDic)

It replaces the other ones and is much better because it's a newer dictionary version

**Comment by Vilhelm-Ian** [comment]
it be nice if extensions like the above one existed for other languages. If someone explained how that addon works I would try to make one for another language

**Comment by mjuhanne** [comment]
@Vilhelm-Ian You could write your own morphimizer - it's pretty easy. You could check [this deaccentmorphimizer](https://github.com/kaegi/MorphMan/commit/3bfef647941acf7f5a0a5a83f3797c1e28c5fb8a) and use it as an example to write your mapping.
